pytest-selenium/pages/content.py

```diff
@@ -217,20 +233,58 @@ def chapter_section(self):
                 return None
 
     class ToolBar(Region):
-        _root_locator = (By.CSS_SELECTOR, '[data-testid="toolbar"]')
+        _root_locator = (By.CSS_SELECTOR, "[data-testid='toolbar']")
         _toc_toggle_button_locator = (
             By.CSS_SELECTOR,
             "[aria-label='Click to open the Table of Contents']",
         )
+        _search_textbox_desktop_locator = (By.CSS_SELECTOR, "[data-testid='desktop-search-input']")
+        _search_button_desktop_locator = (By.CSS_SELECTOR, "button:nth-of-type(2)[value='Search']")
+        _search_button_mobile_locator = (By.CSS_SELECTOR, "[data-testid='mobile-toggle']")
 
         @property
         def toc_toggle_button(self):
             return self.find_element(*self._toc_toggle_button_locator)
 
+        @property
+        def search_textbox(self):
+            return self.find_element(*self._search_textbox_desktop_locator)
+
+        @property
+        # search icon within search text box in desktop
+        def search_button(self):
+            return self.find_element(*self._search_button_desktop_locator)
+
+        @property
+        def search_button_mobile(self):
+            return self.find_element(*self._search_button_mobile_locator)
+
         def click_toc_toggle_button(self):
             self.offscreen_click(self.toc_toggle_button)
             return self.page.sidebar.wait_for_region_to_display()
 
+        def click_search_icon(self):
+            # clicks search icon in mobile
+            self.offscreen_click(self.search_button_mobile)
+
+        def search_for(self, element):
+            self.search_textbox.send_keys(element)
```

Comment:
What is `element` expecting? Maybe add typing to the parameter (ie `element: str`) or a docstring to the function explaining the context and how to use `search_for` with valid parameter values.

Reply:
Added docstring